skimage/restoration/tests/test_denoise.py

```diff
@@ -179,6 +179,25 @@ def test_denoise_tv_bregman_3d():
     assert_(out1[30:45, 5:15].std() > out2[30:45, 5:15].std())
 
 
+def test_denoise_tv_bregman_3d_multichannel():
+    img_astro = astro.copy()
+    denoised0 = restoration.denoise_tv_bregman(img_astro[..., 0], weight=60.0)
+    denoised = restoration.denoise_tv_bregman(img_astro, weight=60.0,
+                                              multichannel=True)
+
```

Comment:
May be can you check that `assert_equal(denoised0, denoised[..., 0])`?

Reply:
Sure. Thank you for pointing this out. I must have missed it.